source/sonata_simulation.rst

```diff
@@ -281,7 +281,7 @@ Cells are held at indicated membrane voltage by injecting adapting current.
    Property                       Type       Requirement  Description
    ============================== ========== ============ ==========================================
    voltage                        float      Mandatory    Specifies the membrane voltage the targeted cells should be held at in mV.
-   rs                             float      Optional     Specifies the series resistance in M :math:`\Omega`. Default is 0.01 M :math:`\Omega`.
+   rs                             float      Optional     Specifies the series_resistance in M :math:`\Omega`. Default is 0.01 M :math:`\Omega`.
```

Comment:
```suggestion
   series_resistance              float      Optional     Specifies the series resistance in M :math:`\Omega`. Default is 0.01 M :math:`\Omega`.
```

Reply:
Yes, the last change that I've pushed has this change in.